src/app/home/home.page.ts

```diff
@@ -85,17 +85,22 @@ export class HomePage {
     this.profileService.getProfileAndEntityConfigData().subscribe(async (mappedIds) => {
       if (mappedIds) {
         this.profilePayload = mappedIds;
-        this.getdata();
+      await  this.getdata();
       }
     });
   }
 
-  getdata() {
+async  getdata() {
+    let  improvements =  `${urlConfig.project.myImprovementsUrl}?&page=${this.page}&limit=${this.limit}&language=en`;
     const urls = {
-      improvements: `${urlConfig.project.myImprovementsUrl}&page=${this.page}&limit=${this.limit}`,
       spotlight: `${urlConfig.project.spotlightUrl}&page=${this.page}&limit=${this.limit}`,
-      recommendation: `${urlConfig.project.recommendationUrl}?page=${this.page}&limit=${this.limit}`,
     };
+    const fetchDataImprovements = (url:any) =>
+      this.baseApiService.post(url,{}).pipe(
+        catchError((err) => {
+          this.toastService.presentToast(err.error.message, 'danger');
```

Comment:
When we show this error msg via toast, will the user be able to understand it? are we sure of it? or should we log this error and show some use friendly msg as toast msg

Reply:
we need to request the msg from the products team